helix-front/package.json

```diff
@@ -30,7 +31,7 @@
     "format": "yarn prettier --write ."
   },
   "engines": {
-    "node": "14.17.6",
+    "node": "~14.17.5",
```

Comment:
With some experimentation, I realized that we can loosen this a little bit to 1) start at patch version 5, 2) accept higher higher patch versions that have the same major and minor version (this is what the `~` means).